v1-preparations/LIST_OF_REMOVED_RESOURCES_FOR_V1.md

```diff
@@ -1,14 +1,10 @@
-Deprecated resources that will be removed with the V1:
+Deprecated resources that are removed with the V1:
 
-* [snowflake_database_old](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/database_old)
-* [snowflake_tag_masking_policy_association](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/tag_masking_policy_association)
-* [snowflake_procedure](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/procedure)
```

Comment:
nit: let's still leave the ones that will be removed as a separate paragraph to remember about them (these files don;t care for the binary but if we remove them and not leave a TODO we might forget about it)

Reply:
Done.